app/helpers/application_helper.rb

```diff
@@ -1,14 +1,24 @@
 module ApplicationHelper
   # TODO: MOVE TO helper in react_on_rails
   # See application.html.erb for usage example
-  def env_javascript_include_tag(prod_asset, dev_asset, params = {})
-    asset_file = !Rails.env.development? ? prod_asset : dev_asset
-    return javascript_include_tag(asset_file, params) if asset_file
+  def env_javascript_include_tag(args = {})
+    send_tag_method(:javascript_include_tag, args)
   end
 
   # TODO: MOVE TO helper in react_on_rails
-  def env_stylesheet_link_tag(prod_asset, dev_asset, params = {})
-    asset_file = !Rails.env.development? ? prod_asset : dev_asset
-    return stylesheet_link_tag(asset_file, params) if asset_file
+  def env_stylesheet_link_tag(args = {})
+    send_tag_method(:stylesheet_link_tag, args)
+  end
+
+  private
+
+  def send_tag_method(tag_method_name, args)
+    asset_type = use_hot_reloading? ? :hot : :static
+    assets = Array(args[asset_type])
+    send(tag_method_name, *assets, args[:options])
```

Comment:
remove the hot and static params from the options,

and let's change this so we don't have to have a special key of :options.

`env_javascript_include_tag(static: ['a', 'b'], turbo: true)

so just take out :static and :hot and pass the rest.